packages/build/tests/plugins/tests.js

```diff
@@ -278,7 +300,15 @@ test('--saveConfig deletes redirects file if redirects were changed', async (t)
   const fixtureRedirectsPath = `${fixtureDir}/_redirects_file`
   const redirectsPath = `${fixtureDir}/_redirects`
   await Promise.all([cpFile(fixtureConfigPath, configPath), cpFile(fixtureRedirectsPath, redirectsPath)])
-  await runFixture(t, 'config_save_redirects', { snapshot: false, flags: { config: configPath, saveConfig: true } })
+  const { address, stopServer } = await startDeployServer()
```

Comment:
`saveConfig` now only happens when deploying, i.e. `buildbotServerSocket` must also be specified.